### DISCUSSION TITLE
Allow name conflicts

### DISCUSSION THREAD

**Discussion by dertuxmalwieder** [tldr-pages/tldr#13670] [Ideas]
I would like to document the `sam` editor, but I noticed that `sam` is already used by a less common command on TLDR. Now I don't know what to do. A solution would be good.

**Comment by TheRootDaemon** [comment]
You could place it in `pages/common/sam.md`, I don't think there is one yet in the common pages.

However the AWS's sam.md is inside of `pages/linux/sam.md`

**Comment by spageektti** [comment]
> You could place it in `pages/common/sam.md`, I don't think there is one yet in the common pages.
> 
> However the AWS's sam.md is inside of `pages/linux/sam.md`

Do not do that. We can move the page to `aws-sam.md` or something like that and make the `sam.md` an alias page referencing both commands
